pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -850,11 +850,11 @@ func (m *kubeGenericRuntimeManager) computePodActions(ctx context.Context, pod *
 	}
 
 	// Check initialization progress.
```

Comment:
Clarify in this comment that this still can find work (sidecars to restart) even post-initialization?

Reply:
done.